File: year2/systems-design-implementation/Lab4/Lab4Sharp/Domain/Proba.cs
using Lab2Sharp.Utils;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab2Sharp.Domain
{
    public class Proba : Entity<int>
    {
        public Proba(int id, int distanta, string stil)
        {
            base.id = id;
            this.distanta = distanta;
            this.stil = stil;
        }

        public String stil { get; set; }
        public int distanta { get; set; }

        public override string ToString()
        {
            return stil + " " + distanta + "m";
        }
    }
}

File: year2/systems-design-implementation/Lab3/Lab3/src/main/java/domain/Proba.java
package domain;

import utils.Entity;

public class Proba extends Entity<Integer> {

    private Integer distanta;
    private String stil;

    public Proba(Integer id, Integer distanta, String stil) {
        super(id);
        this.distanta = distanta;
        this.stil = stil;
    }

    public Integer getDistanta() {
        return distanta;
    }

    public void setDistanta(Integer distanta) {
        this.distanta = distanta;
    }

    public String getStil() {
        return stil;
    }

    public void setStil(String stil) {
        this.stil = stil;
    }

    @Override
    public String toString() {
        return super.toString() + " Proba{" +
                "distanta=" + distanta +
                ", stil='" + stil + '\'' +
                '}';
    }
}
